tools/src/test/java/org/apache/kafka/tools/MetadataQuorumCommandTest.java

```diff
@@ -53,12 +53,12 @@ public MetadataQuorumCommandTest(ClusterInstance cluster) {
      * 3. Fewer brokers than controllers
      */
     @ClusterTests({
-        @ClusterTest(clusterType = Type.CO_KRAFT, brokers = 3, controllers = 3),
-        @ClusterTest(clusterType = Type.KRAFT, brokers = 3, controllers = 3),
-        @ClusterTest(clusterType = Type.CO_KRAFT, brokers = 3, controllers = 2),
-        @ClusterTest(clusterType = Type.KRAFT, brokers = 3, controllers = 2),
-        @ClusterTest(clusterType = Type.CO_KRAFT, brokers = 2, controllers = 3),
-        @ClusterTest(clusterType = Type.KRAFT, brokers = 2, controllers = 3)
+        @ClusterTest(clusterType = Type.CO_KRAFT, brokers = 1, controllers = 1),
+        @ClusterTest(clusterType = Type.KRAFT, brokers = 1, controllers = 1),
```

Comment:
The test name indicates it's testing replication, that doesn't happen if there's a single broker & controller, right?

Reply:
Good point! Let me use 2 + 2 nodes instead